### PR TITLE
DR | Fix review page styling

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > - The area of disagreement page for Notice of Disagreement & Higher-Level Review show an update page button without appropriate margins. It's fixed by this PR
  > - When an accordion has unviewed pages within it, the accordion header, page sub-header and border change color to emphasize that the content needs attention. These have been accidentally removed after updating to use USWDS v3 web components, and changing variables in SCSS and CSS variables
- _(If bug, how to reproduce)_
  > - Navigate to NOD or HLR's review & submit page
  > - Unselect then reselect all selected issues to be reviewed
  > - The area of disagreement section will show an error icon in the sub-header and you'll see the accordion header also include an icon
  > - There are no color changes to the header or subheader
  > - And, if you open the dev tools and target the accordion toggle button, add a background color and you'll see that the update button has no bottom margin
- _(What is the solution, why is this the solution)_
  > - Add update page top & bottom margins
  > - Fix accordion header & sub-header color definitions - the header will be fixed after updating the component library
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#77061](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77061)

## Testing done

- _Describe what the old behavior was prior to the change_
  > No margin around "Update page" in the area of disagreement component
  > Missing sub-header background color styling
- _Describe the steps required to verify your changes are working as expected_
  > Test in review instance
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Update page button | &dagger; character count text, update button and header of the next area of disagreement page with minimal vertical spacing between them | Screenshot 2024-03-28 at 9 01 38 AM |
| Subheader background | Two disagreement subheaders with no background styling | Two disagreement subheaders with light red background |

&dagger; Note that the Update page button "before" screenshot was before taken before the subheader styling was accidentally removed.

## What areas of the site does it impact?

- Update page margins - NOD & HLR
- Sub-header styling - All forms that use the review & submit page

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/platform/forms/sass/_m-schemaform.scss [hunk 1]
```diff
@@ -535,18 +535,14 @@ legend.schemaform-label.schemaform-file-label {
...
 va-accordion-item[data-unviewed-pages="true"] {
-  --item-border: 3px solid var(--color-secondary-lightest) !important;
-
   &::part(accordion-header) {
-    background-color: var(--color-secondary-lightest);
-    border-color: var(--color-secondary-lightest);
+    background-color: #{$color-secondary-lightest};
+  }
+  &::part(accordion-content) {
+    border-color: #{$color-secondary-lightest};
```
Comment (Mottie) [inline src/platform/forms/sass/_m-schemaform.scss:544]: Following "will work in all Sass versions" [recommendations from the documentation](https://sass-lang.com/documentation/breaking-changes/css-vars/)

File: src/platform/forms/sass/_m-schemaform.scss [hunk 2]
```diff
@@ -535,18 +535,14 @@ legend.schemaform-label.schemaform-file-label {
...
   }
-}
-
-.form-review-panel .accordion-header > button.usa-button-unstyled {
-  padding: 2.5rem 4.5rem 2.5rem 2.5rem;
-  line-height: 2.2rem;
 }
```
Comment (Mottie) [inline src/platform/forms/sass/_m-schemaform.scss:546]: `.accordion-header` existed in the HTML accordion (non-web component) version

Comment (rocio-desantiago) [reply src/platform/forms/sass/_m-schemaform.scss:546]: ah nice. Thanks for that context.

File: src/applications/appeals/shared/components/AreaOfDisagreement.jsx [hunk 3]
```diff
@@ -159,7 +159,12 @@ const AreaOfDisagreement = ({
       </VaCheckboxGroup>
 
       {onReviewPage ? (
-        <va-button text={content.update} onClick={handlers.updatePage} uswds />
+        <va-button
+          class="vads-u-margin-top--2 vads-u-margin-bottom--4"
+          text={content.update}
+          onClick={handlers.updatePage}
+          uswds
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/shared/components/AreaOfDisagreement.jsx:166]: I started to, but figured that's something we can do to all 3 forms at once in the future.

**Review by rocio-desantiago — CHANGES_REQUESTED** [review]
uswds prop?

**Review by rocio-desantiago — APPROVED** [review]

File: src/platform/forms/sass/_m-schemaform.scss (hunk 1 above)
Comment (Mottie) [reply src/platform/forms/sass/_m-schemaform.scss:544]: These changes, along with this [component library update](https://github.com/department-of-veterans-affairs/component-library/pull/1098), will fix the accordion highlighting on the review & submit page

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**